x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/util/Check.java

```diff
@@ -36,4 +36,14 @@ public static void notNull(Object object, String message, Object... values) {
             throw new SqlIllegalArgumentException(message, values);
         }
     }
+    
+    public static void isNumberOutOfRange(Object object, String objectName, Long from, Long to) {
+        if (!(object instanceof Number)) {
```

Comment:
```suggestion
        if ((object instanceof Number) == false) {
```
This is a syntax part of [CONTRIBUTING.md](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md). Not everybody loves it, but it is more obvious.

Reply:
Thanks for the advice.